protocol_rfcs/collated-string-type.md

```diff
@@ -8,12 +8,33 @@ This protocol change adds support for collated strings. It consists of three cha
 * Domain metadata with active collation version
 
 --------
+> *** Add New Section after the [Clustered Table](#clustered-table) section***
+# Collations Table Feature
 
-> ***Add a new section in front of the [Primitive Types](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#primitive-types) section.***
+To support this feature:
+* The table must have Writer Version 7. A feature name `collations` must exist in the table's `writerFeatures`.
+* The feature `domainMetadata` is required in the table protocol's `writerFeatures`. 
+
+## Reader Requirements for Collations:
+
+When Collations is supported (when the `writerFeatures` field of a table's protocol action contains `collations`), then:
+- Readers could do comparisons and sorting of strings based on the collation specified in the schema. The choice of collation and collation version is up to the reader.
+- If the collation is not specified for a string type, then the reader must use the default utf8 binary collation.
 
-### Collations table feature
+## Writer Requirements for Collations:
+
+When Collations is supported (when the `writerFeatures` field of a table's protocol action contains `collations`), then:
+- Writers must write the collation identifier in the schema metadata for a column with non-default collation (utf8 binary).
+- Writer must not write the collation identifier in the schema metadata for a column with default collation (utf8 binary).
+- Writers could write per-file statistics for string columns with collations other than utf8 binary collation in `statsWithCollation`. See [Per-file Statistics](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#per-file-statistics) for more details.
+- Writers could collect per-file statistics for collation versions that are specified in the `domainMetadata` for the `collations` table feature.
```

Comment:
I like the idea to keep it flexible, but does it still make sense to add what is a good default behaviour? I guess this is only using the latest version. If the "default" is to collect for all, we can quickly blow up stats size. While this is fine to be allowed, I think a hint on what is a good default to implement makes sense here.